Mathlib/Order/Fin/Monotone.lean

```diff
@@ -0,0 +1,75 @@
+/-
+Copyright (c) 2025 Joël Riou. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Joël Riou
+-/
+import Mathlib.Order.Fin.Basic
+
+/-!
+# Monotone maps from `Fin`
+
+In this file, we show that a map `f : Fin (n + 1) → α` is monotone
+iff `f i.castSucc ≤ f i.succ` for all `i : Fin n`.
+
+-/
+
+universe u
+
+namespace Fin
+
+lemma monotone_iff {α : Type u} [Preorder α] {n : ℕ} (f : Fin (n + 1) → α) :
```

Comment:
isnt' this https://leanprover-community.github.io/mathlib4_docs/Mathlib/Order/Fin/Basic.html#Fin.monotone_iff_le_succ ?

Reply:
Thanks very much!